scripts/update_swagger.sh

```diff
@@ -16,6 +16,12 @@ cat ${filename} | \
     'del(.paths[][]."responses"|select(has("400"))."default" ) | del(.paths[][]."responses"|select(has("201"))."200") | del(.paths[][]."responses"|select(has("204"))."200")' > ${tmp_filename}
 mv ${tmp_filename} ${filename}
 
+# Add an example value to the ConsistencyPreference to override the default of UNSPECIFIED being shown in the docs
+cat ${filename} | \
+  jq \
+    '.definitions.ConsistencyPreference.example = "MINIMIZE_LATENCY"' > ${tmp_filename}
+mv ${tmp_filename} ${filename}
```

Comment:
I don't love this, but I'm not able to override the example of UNSPECIFIED any other way; because we use a ref (as we should) anything specified where used won't have any effect, e.g., `ConsistencyPreference consistency = 8 [(validate.rules).enum.defined_only = true];`. If you open the generated definition in a swagger editor, you will see warnings about just that.

Reply:
For the record, there's an open ticket to allow this https://github.com/grpc-ecosystem/grpc-gateway/issues/2671